NextcloudTalk.xcodeproj/project.pbxproj

```diff
@@ -2727,8 +2740,8 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/nextcloud/ios-communication-library";
 			requirement = {
-				kind = exactVersion;
-				version = 0.98.0;
+				branch = master;
+				kind = branch;
```

Comment:
I can't build the app using NCComm. master:
```
NextcloudTalk/NCAPIController.m:104:37: No visible @interface for 'NCCommunicationCommon' declares the selector 'setupWithAccount:user:userId:password:urlBase:userAgent:webDav:dav:nextcloudVersion:delegate:'```
among others.. on first glance looks like it doesn't like default paramter values

Reply:
Yeah, looks like the NCCommunication API has changed in master and the tag for the release set by default is no longer there... I made some local changes to fix the compile issues but they are kind of out of scope, I can open a new PR with them

Reply:
Is there a reason to move to master branch (or any other version) for NCComm? If not I don't think changes to the dependencies should be done in this PR.

Reply:
I was having issues pulling the 0.98.0 package in Xcode though it seems that was all local because it works now. I will revert the changes and open a new PR adding compatibility with the latest version of the package

Reply:
Reverted